application/dash_application/dash_routes.py

```diff
@@ -13,8 +13,8 @@ def add_dash(server):
         page.init_callbacks(dash_app)
 
     ## Set your initial Dash page's layout to this variable 
-    from . import url_handler
-    dash_app.layout = url_handler.layout
+    from .views import index_page
```

Comment:
It seems like changing . to .views is the whole of the fix here. I'm not sure I agree with changing 'url_handler' to 'index_page' because that makes it sound like it plays the role of 'application/templates/index.html'. url_handler/index_page doesn't actually have any content of its own it displays

Reply:
Hrm, looking at it more closely it does look like a router for known pages, as opposed to a view. Having the layout element tricked me into thinking that it would display something. It sounds like it could make sense to rename `url_handler/index_page` to `known_pages_router` and to clearly comment what its function is (and keep it out of the `views` folder). Sound ok @avarsava ?

Reply:
I thought `url_handler` described what it did (take action based on incoming URL values), but i'm also OK with `known_pages_router` if you think that makes more sense :) and more documentation is always good

Reply:
Does it handle all URL-related changes, like query strings or anchors?

Reply:
it _could_ , by virtue of all Dash callbacks existing in memory at once, but that's not really the intended pattern

Reply:
Then I will update it to `known_pages_router` since it's not intended to handle all URL-related changes, since that's an important disambiguation in my brain and is acceptable to yours :)